tests/Feature/OrganisationTest.php

```diff
@@ -1410,6 +1410,33 @@ public function test_the_application_cannot_update_organisations_approved(): voi
         $this->assertEquals('Invalid argument(s)', $message);
     }
 
+    public function test_the_application_update_organisations_approved_by_admin(): void
+    {
+        $organisationIdTest = 1;
+        $initSystemApproved = Organisation::where('id', $organisationIdTest)->first()->system_approved;
+
+        if ($initSystemApproved === true) {
+            Organisation::where('id', $organisationIdTest)->update(['system_approved' => 0]);
```

Comment:
This is suppose to be boolean or tinyInt? Is defined as `0` and later there is a check assertEquals `true` - mixing types and is auto-casting in tests?

Reply:
I think this is happening because we are `casting` on this field in the model.